certbot/plugins/manual_test.py

```diff
@@ -81,16 +77,14 @@ def test_script_perform(self):
             'print(os.environ.get(\'CERTBOT_TOKEN\', \'notoken\'));'
             'print(os.environ.get(\'CERTBOT_CERT_PATH\', \'nocert\'));'
             'print(os.environ.get(\'CERTBOT_KEY_PATH\', \'nokey\'));'
```

Comment:
Can we remove the lines for `CERTBOT_CERT_PATH` and `CERTBOT_KEY_PATH`?

Reply:
Done.